DALI.tex

```diff
@@ -948,40 +954,57 @@ \subsection{Shape}
 more limited context.
 
 \subsection{URI}
-URI values \citep{std:RFC3986} serialised in VOTable or service parameters 
-should have the following metadata in the \xmlel{FIELD} element: \verb|datatype="char"|, 
-\verb|arraysize="*"|, \verb|xtype="uri"| (where arraysize may also be fixed length or 
+URI values \citep{std:RFC3986} serialised in VOTable or service parameters
+should have the following metadata in the \xmlel{FIELD} element: \verb|datatype="char"|,
+\verb|arraysize="*"|, \verb|xtype="uri"| (where arraysize may also be fixed length or
 variable length with limit).
 
 \subsection{UUID}
-Universal Unique Identifier (UUID) values serialised in VOTable or service parameters 
-should have the following metadata in the \xmlel{FIELD} element: \verb|datatype="char"|, 
-\verb|arraysize="36"|, \verb|xtype="uuid"| (where arraysize may also be fixed length or 
+Universal Unique Identifier (UUID) values serialised in VOTable or service parameters
+should have the following metadata in the \xmlel{FIELD} element: \verb|datatype="char"|,
+\verb|arraysize="36"|, \verb|xtype="uuid"| (where arraysize may also be fixed length or
 variable length with limit).
 
-UUID values \citep{std:RFC4122} are serialised using the canonical ascii (hex) 
+UUID values \citep{std:RFC4122} are serialised using the canonical ascii (hex)
 representation, for example: e0b895ca-2ee4-4f0f-b595-cbd83be40b04.
 
+\subsection{JSON}
+
+For complex data blobs that are not intended to be queriable, publishers
+may choose to store them as serialised JSON \citep{std:JSON} strings in
+VOTables.  Note that in general, an appropriate VOTable serialisation
+(e.g., as a \xmlel{GROUP} for a \xmlel{PARAM} or as unrolled
+\xmlel{FIELD}\/s or a separate table for table cells) is preferred.
+
+\xmlel{PARAM}\/s and \xmlel{FIELD}\/s containing JSON values should be
+marked up with a \xmlel{datatype} of \verb|char| or \verb|unicodeChar|,
+with variable or fixed \xmlel{arraysize} and an \xmlel{xtype} of
+\verb|json|.
+
+Libraries supporting the json xtype should return deserialised sequences
+and mappings as appropriate for the host language for json payloads.
```

Comment:
I don't know whether this comment is necessary or beneficial.  The other xtypes don't have anything similar, for instance timestamp doesn't say that libraries should return a language-specific timestamp object.  I'd be inclined to delete this paragraph, but if not I would prefer "may" to "should".